# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nas/X11R7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- nas-1.7/Makefile	2002-01-20 20:51:24.000000000 +0100
+++ nas-1.7-fixed/Makefile	2005-11-04 15:40:22.000000000 +0100
@@ -71,11 +71,11 @@
          IMAKESRC = $(CONFIGSRC)/imake
         DEPENDSRC = $(CONFIGSRC)/makedepend
 
-          INCROOT = /usr/X11R6/include
-        USRLIBDIR = /usr/X11R6/lib
-         SHLIBDIR = /usr/X11R6/lib
+          INCROOT = /usr/X11/include
+        USRLIBDIR = /usr/X11/lib
+         SHLIBDIR = /usr/X11/lib
        LINTLIBDIR = $(USRLIBDIR)/lint
-          MANPATH = /usr/X11R6/man
+          MANPATH = /usr/X11/man
     MANSOURCEPATH = $(MANPATH)/man
            MANDIR = $(MANSOURCEPATH)1
         LIBMANDIR = $(MANSOURCEPATH)3
@@ -171,7 +171,7 @@
      INSTDATFLAGS = -m 0444
     INSTKMEMFLAGS = -s -m 4711
 
-      PROJECTROOT = /usr/X11R6
+      PROJECTROOT = /usr/X11
 
       CDEBUGFLAGS = -O3 -mpentium -mieee-fp -fbuiltin
         CCOPTIONS = -pipe
@@ -223,7 +223,7 @@
 # X Window System make variables; these need to be coordinated with rules
 
              XTOP = $(XPROJECTROOT)
-           BINDIR = /usr/X11R6/bin
+           BINDIR = /usr/X11/bin
      BUILDINCROOT = $(TOP)/exports
       BUILDINCDIR = $(BUILDINCROOT)/include
       BUILDINCTOP = ../..
@@ -286,7 +286,7 @@
      TRANSCOMMSRC = $(LIBSRC)/xtrans
    TRANS_INCLUDES = -I$(TRANSCOMMSRC)
 
-     XPROJECTROOT = /usr/X11R6
+     XPROJECTROOT = /usr/X11
 
        XENVLIBDIR = $(USRLIBDIR)
    CLIENTENVSETUP = XLOCALEDIR=$(XBUILDINCROOT)/lib/locale LD_LIBRARY_PATH=$(DTENVLIBDIR):$(OGLENVLIBDIR):$(MOTIFENVLIBDIR):$(XENVLIBDIR)
@@ -492,7 +492,7 @@
 
     MOTIFENVLIBDIR = $(USRLIBDIR)
 
-      USRINCDIR = /usr/X11R6/include
+      USRINCDIR = /usr/X11/include
          UIDDIR = $(LIBDIR)/uid
         TESTSRC = $(MTOP)/tests
         TESTLIB = $(TESTSRC)/lib
@@ -524,7 +524,7 @@
    MRESOURCESRC = $(MLIBSRC)/Mrm
          UILSRC = $(MCLIENTSRC)/uil
 
-   MPROJECTROOT = /usr/X11R6
+   MPROJECTROOT = /usr/X11
 
             UIL = uil
          DEPUIL = $(BINDIR)/uil
@@ -584,7 +584,7 @@
 
    OGLENVLIBDIR = OBuildLibPath
 
-   OPROJECTROOT = /usr/X11R6
+   OPROJECTROOT = /usr/X11
 
 SOGLREV = 1.1
 DEPGLLIB =
